### PR TITLE
Move the repo to the latest Roslyn bits (signed build from yesterday night).

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

File: src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/InterfaceMethodsShouldBeCallableByChildTypes.cs [hunk 1]
```diff
@@ -66,16 +66,16 @@ private static bool ShouldExcludeOperationBlock(ImmutableArray<IOperation> opera
             {
                 IBlockOperation block = operationBlocks[0] as IBlockOperation;
 
-                // An operation block that's not even a block - don't analyze the error cases.
+                // Analyze IBlockOperation blocks.
+                // TODO: Handle expression bodied members https://github.com/dotnet/roslyn-analyzers/issues/1345
```
Comment (333fred) [inline src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/InterfaceMethodsShouldBeCallableByChildTypes.cs]: I thought we had expression-bodied members be blocks as well, from the IOperation standpoint.

File: src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb [hunk 2]
```diff
@@ -57,14 +57,14 @@ Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.Maintainability
 
                         operationBlockContext.RegisterOperationAction(
                         Sub(operationContext)
-                            Dim declarations = DirectCast(operationContext.Operation, IVariableDeclarationsOperation).Declarations
+                            Dim declarations = DirectCast(operationContext.Operation, IVariableDeclarationGroupOperation).Declarations
```
Comment (333fred) [inline src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb]: We can simplify this with [https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L102](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L102)

File: src/Microsoft.NetFramework.Analyzers/Core/DoNotUseInsecureDtdProcessing.cs [hunk 3]
```diff
@@ -630,9 +630,9 @@ private void AnalyzeObjectCreationForXmlReaderSettings(OperationAnalysisContext
             private void AnalyzeVariableDeclaration(OperationAnalysisContext context)
             {
                 var declare = (IVariableDeclarationOperation)context.Operation;
-                foreach (var variable in declare.Variables)
+                foreach (var declarator in declare.Declarators)
                 {
-                    AnalyzeObjectCreationInternal(context, variable, declare.Initializer?.Value);
+                    AnalyzeObjectCreationInternal(context, declarator.Symbol, declarator.Initializer?.Value ?? declare.Initializer?.Value);
```
Comment (333fred) [inline src/Microsoft.NetFramework.Analyzers/Core/DoNotUseInsecureDtdProcessing.cs]: Declarator has an extension method that will do the right thing here: [https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L147](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L147)

**Comment by 333fred** [comment]
Done with review Iteration 2

**Comment by tmat** [comment]
test this please

**Comment by mavasani** [comment]
@333fred Addressed your feedback

File: src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb (hunk 2 above)
Comment (333fred) [reply src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb]: We don't even have to have the nested for loops, the extension I linked operates on IVariableDeclarationGroupOperation.

---
In reply to: [150135540](https://github.com/dotnet/roslyn-analyzers/pull/1346#discussion_r150135540) [](ancestors = 150135540)

**Comment by mavasani** [comment]
@dotnet-bot test this please

**Pull request merged**